lib/pages/App.js

```diff
@@ -173,12 +213,43 @@ export default class App extends Component {
             targetId={uploadTargetUser}
             timezoneIsSelected={Boolean(this.props.selectedTimezone)}
             userDropdownShowing={this.props.showingUserSelectionDropdown}
+            isClinicAccount={this.props.isClinicAccount}
             userIsSelected={uploadTargetUser !== null}
             addDevice={this.props.sync.addTargetDevice}
             removeDevice={this.props.sync.removeTargetDevice}
             onDone={this.props.async.clickDeviceSelectionDone} />
         </div>
       );
+    } else if (page === pages.CLINIC_USER_SELECT) {
+      const { allUsers, page, targetUsersForUpload, uploadTargetUser } = this.props;
+      return (
+        <div>
+          <ClinicUserSelect
+            allUsers={allUsers}
+            onUserChange={this.props.async.setUploadTargetUserAndMaybeRedirect}
```

Comment:
[This comment](https://github.com/tidepool-org/chrome-uploader/pull/294#discussion_r67573104) hasn't been addressed yet, it looks like. If we discussed and decided not to change anything, remind me, cuz I don't remember.